groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/completion/KeywordCompletion.java

```diff
@@ -98,6 +110,27 @@ public boolean complete(Map<Object, CompletionProposal> proposals, CompletionCon
         return true;
     }
 
+    boolean isFirstStatement(final CompletionContext request) {
+        TokenSequence<GroovyTokenId> ts = LexUtilities.getGroovyTokenSequence(request.doc, 1);
+
+        if (ts != null) {
+            ts.move(request.lexOffset);
+            if (ts.movePrevious()) {
+                while (ts.isValid() && ts.movePrevious() && ts.offset() >= 0) {
```

Comment:
Nitpick: IMHO ts.offset() >= 0 is redundant on all places -- if `movePrevious` had failed bcs there's no previous token, it would return `false`; and the 1st token is always at offset 0.

Reply:
I wanted to be consistent with the rest of the code. Else this should be changed in more places, not only in my patch.